src/main/java/com/mysticalchemy/api/events/CrucibleEvent.java

```diff
@@ -0,0 +1,92 @@
+package com.mysticalchemy.api.events;
+
+import com.mysticalchemy.recipe.PotionIngredientRecipe;
+import net.minecraft.world.effect.MobEffect;
+import net.minecraft.world.entity.LivingEntity;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.item.ItemStack;
+import net.minecraftforge.eventbus.api.Cancelable;
+import net.minecraftforge.eventbus.api.Event;
+
+import java.util.HashMap;
+
+/**
+ * This event is fired when the crucible is used.
+ */
+public class CrucibleEvent extends Event {
+
+    public CrucibleEvent() {}
```

Comment:
The empty ctor is meaningless; it can be added if there is actual common functionality that should be initialized.